core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.html

```diff
@@ -1,4 +1,18 @@
 <div class="texera-workspace-operator-panel-body">
+  <div class="texera-workspance-search-box">
+    <form class="search-form">
+      <mat-form-field class="example-full-width">
+        <input class="form-control" type="text" placeholder="Search" aria-label="Search"
+        matInput [matAutocomplete]="auto" [formControl]="myControl">
+        <mat-autocomplete #auto="matAutocomplete">
+          <mat-option (click)="OnHumanSelected(option)" *ngFor="let option of filteredOptions | async" [value]="option" >
+            {{option}}
+          </mat-option>
+          <!-- <i class="fa fa-search" aria-hidden="true"></i> -->
+        </mat-autocomplete>
+      </mat-form-field>
+    </form>
+  </div>
```

Comment:
Change class name `example-full-width` to something like `operator-search-form`

On the Angular Material page for autocomplete, I realize that there is a group feature for the auto-complete:
https://material.angular.io/components/autocomplete/overview

You can try to implement this. For the group name, just use the group we have for each operator (operatorGroupName)